src/StripeResource.ts

```diff
@@ -157,6 +157,8 @@ StripeResource.prototype = {
     const options = getOptionsFromArgs(args);
     const host = options.host || spec.host;
     const streaming = !!spec.streaming;
+    const apiMode = options.apiMode || 'standard';
```

Comment:
Should we support overriding apiMode for non raw_request calls?

Reply:
Ah you're right, we shouldn't let this leak into non raw_request calls. Let me refactor this